modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -49,42 +49,49 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
+** If the toggle is off, endpoints that use SSL certificates may be self-signed, but do not have to be.
++
+To test a combination of endpoints that use trusted SSL certificates and endpoints that use self-signed certificates, create two separate monitors: one for testing the first group of endpoints, the other for testing the second group of endpoints.
+. Click *Next*.
+. In the *Endpoints* section, specify one or more endpoints to include in the test, the HTTP methods to use to test them, the status codes that each endpoint is expected to return, and any assertions that you want to include for each endpoint.
+. Click *Next*.
+. If you want to be notified of failed tests, select one or more reporters.
+. Click *Create monitor*.
 
 == Result
 
-The dashboard opens. The monitor that you created is listed on the left.
-
-Beneath the name of the monitor is an indicator that explains when the current run of the monitor began. A line at the bottom of the list item shows how much time remains before the next run of the monitor. Also, the *Monitored Endpoints*, *Tests*, and *Schedules* sections will all be empty.
-
-After the monitor has run, the list item for the test suite will indicate whether the monitored endpoints passed or failed.
+The monitor that you created is listed at the top of the list of your monitors. API Functional Monitoring begins counting down the interval of time that you selected. When the interval expires, the test is run.
 
 [IMPORTANT]
 ====
-Tests silently fail when they take more than 120 seconds to run. As a workaround, you can split a long-running test into multiple tests. Download the monitor, split the test across multiple monitors in the BAT CLI, run and time each test to be sure that it is under 120 seconds, and then upload the monitors.
+Tests fail when they take more than 120 seconds to run. As a workaround, you can split a long-running test into multiple tests. Download the monitor, split the test across multiple monitors in the BAT CLI, run and time each test to be sure that it is under 120 seconds, and then upload the monitors.
 
 If you do not want to split a long-running test, download the monitor and run it from the BAT CLI.
 ====
 
-== What to do next
-
-Click the *Scheduling* tab to see the schedule for the monitor. You can add additional schedules and delete schedules that you no longer want to run. You can also disable schedules, if you want to pause them, and then enable them again.
-
-After the monitor runs, click the *Monitored Endpoints* tab to see:
-
-* Whether all endpoints returned the expected codes, as indicated by *PASSED* in the first column of the table
-* Statistics about the execution of the test of the endpoint
-
-As your monitor continues to run at the scheduled intervals, the statistics in this table are updated. They show the minimum, average, and maximum runtime for up to five executions of the test.
+After the test is complete, two things happen:
```

Comment:
Is this strictly true? Two things happen after the first test run; only one thing happens subsequently.